### ISSUE TITLE
Create image upload process

### ISSUE THREAD

**Issue by dchiller** [DDMAL/VIM#106]
Create an image upload process that supports PNG images

**Comment by ahankinson** [comment]
Will images stored elsewhere on the internet be allowed? Can I provide a URL to an image on the internet archive or Wikimedia commons instead of uploading an image?

**Comment by fujinaga** [comment]
Yes, as long as the images are in the public domain, in fact, Dylan had suggested that we may store the images on IA or Wikimedia (see #15).
